autoPyTorch/api/base_task.py

```diff
@@ -1245,17 +1239,32 @@ def __del__(self) -> None:
         # until the estimator is deleted
         self._backend.context.delete_directories(force=False)
 
-    @typing.no_type_check
     def get_incumbent_results(
-            self
-    ):
-        pass
+        self,
+        include_traditional: bool = False
+    ) -> Tuple[Configuration, Dict]:
+        """
+        Get Incumbent config and the corresponding results
+        Args:
+            include_traditional:
 
-    @typing.no_type_check
-    def get_incumbent_config(
-            self
-    ):
-        pass
+        Returns:
+
+        """
+        assert self.run_history is not None, "No Run History found, search has not been called."
+        assert not self.run_history.empty(), "Run History is empty."
+
+        run_history_data = self.run_history.data
+        if not include_traditional:
+            run_history_data = dict(
+                filter(lambda elem: elem[1].additional_info is not None and 'trainer_configuration' not in elem[1].
```

Comment:
Hmm, what does having the key `trainer_configuration `mean in this scenario ? Since when include_traditional is activated, we do not filter the `run_history_data`.

Reply:
so this is an ad-hoc way of checking if the run history belongs to the traditional pipeline or an autopytorch pipeline. I'll add a comment before this to make it clearer

Reply:
Aha, oki, perfect.